nybruker/process.sh

```diff
@@ -94,6 +93,22 @@ if [ $USEREXISTS -eq 1 ]; then
 		done
 	fi
 	
+	OLD_FIRSTNAME=$(echo "$DATA" | grep "^givenName:" | awk '{print $2}' | head -n 1)
+	OLD_LASTNAME=$(echo "$DATA" | grep "^sn:" | awk '{print $2}' | head -n 1)
+	if [[ ("$FIRSTNAME" != "" && "$LASTNAME" != "") && ( "$OLD_FIRSTNAME" != "$FIRSTNAME" || "$OLD_LASTNAME" != "$LASTNAME" ) ]]; then
+
+		echo "Oppdatere navn fra $OLD_FIRSTNAME $OLD_LASTNAME til $FIRSTNAME $LASTNAME?"
+		select yn in "Ja" "Nei"; do
+			if [[ "$yn" == "Ja" ]]; then
+				addchange "givenName" "$FIRSTNAME"
```

Comment:
Det er fint å beholde dette så det blir konsistent med den tidligere koden, men merk at i `bash` så har disse `"` rundt enkle tekst-strenger ingen reell effekt.

`addchange givenName "$FIRSTNAME"` er det samme
Samme med `select yn in Ja Nei; do`

Men behold det gjerne som du har gjort her. Men bærer bare preg at jeg skrev dette uten all kunnskapen jeg har i dag :p